src/cdk/constructs/cicd/deployment-permissions/index.ts

```diff
@@ -12,7 +12,7 @@ export class DeploymentPermissions extends Stack {
   public role: Role
 
   constructor (parent: Construct, props: DeploymentPermissionsProps = { env: 'dev' }) {
-    const name = getBaseStackName('CICDPermissions')
+    const name = `${MiraConfig.getBaseStackName('CICDPermissions')}-${props.env}`
```

Comment:
Just noting from yesterday's call, this is an area where I had mentioned we can abstract away naming.

Reply:
Yep, we probably need an improvement in the Config/Object class. And a well defined docs with rules on how to retrieve values from the config.json file